apps/chameleon/app/(auth)/register/page.tsx

```diff
@@ -1,44 +1,138 @@
+          buttonVariants({ variant: 'aware' }),
           'absolute right-4 top-4 md:right-8 md:top-8'
         )}
       >
         Login
       </Link>
-      <div className="hidden h-full bg-muted lg:block" />
+
+      <div className="relative hidden h-full bg-muted lg:block">
+        <CarouselRegister />
+        {/* <Carousel plugins={[plugin.current]} className="bg-white w-full h-full">
+          <CarouselContent className="w-full h-full">
+            <CarouselItem className="basis-full">
+              <Card>
+                <CardContent className="h-full w-full relative">
+                  <Image
+                    alt="Photo of Cosplayer"
+                    className="p-8"
+                    layout="fill"
+                    src={cosplayer1}
+                  />
+                </CardContent>
+              </Card>
+            </CarouselItem>
+
+            <CarouselItem className="basis-full relative">
+              <Card>
+                <CardContent>
+                  <Image
+                    alt="Photo of Cosplayer"
+                    className="p-8"
+                    layout="fill"
+                    src={photographer1}
+                  />
+                </CardContent>
+              </Card>
+            </CarouselItem>
+
+            <CarouselItem className="basis-full relative">
+              <Card>
+                <CardContent>
+                  <Image
+                    alt="Photo of Cosplayer"
+                    className="p-8"
+                    layout="fill"
+                    src={cosplayer2}
+                  />
+                </CardContent>
+              </Card>
+            </CarouselItem>
+
+            <CarouselItem className="basis-full relative">
+              <Card>
+                <CardContent>
+                  <Image
+                    alt="Photo of Cosplayer"
+                    className="p-8"
+                    layout="fill"
+                    src={photographer2}
+                  />
+                </CardContent>
+              </Card>
+            </CarouselItem>
+
+            <CarouselItem className="basis-full relative">
+              <Card>
+                <CardContent>
+                  <Image
+                    alt="Photo of Cosplayer"
+                    className="p-8"
+                    layout="fill"
+                    src={cosplayer3}
+                  />
+                </CardContent>
+              </Card>
+            </CarouselItem>
+          </CarouselContent>
+        </Carousel> */}
+      </div>
+
       <div className="lg:p-8">
-        <div className="mx-auto flex w-full flex-col justify-center space-y-6 sm:w-[350px]">
-          <div className="flex flex-col space-y-2 text-center">
-            <Icons.logo className="mx-auto h-6 w-6" />
-            <h1 className="text-2xl font-semibold tracking-tight">
+        <div className="mx-auto flex w-full flex-col justify-center space-y-6 sm:w-[350px] text-[#191919]">
```

Comment:
we need to avoid text color arb values. these need to be brought into the design system color theme and then re written as text-insert-theme-color

Reply:
I added the arb values to the theming, thoughts on changing these values to the default black and white in order to help with eye-strain?
![AddedColorsToThemeJPG](https://github.com/Ionia-Devs/chameleon/assets/100187027/f2bee35f-ffce-461e-a570-e1e5d73c2449)

Reply:
Lets remove these and use tailwind default color theme